.github/ISSUE_TEMPLATE/support.yaml

```diff
@@ -0,0 +1,108 @@
+name: 🚢 Support
+description: Support needed with Prometheus Operator
+labels: ["kind/support", "needs-triage"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Note:** This repository is about prometheus-operator itself, if you have questions about:
+            - kube-prometheus setup, go to [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus)
+            - helm installation, go to [kube-prometheus-stack-helm-chart](https://github.com/prometheus-community/helm-charts)
+- type: markdown
+  attributes:
+    value: |
+      Before filing a request, please be sure you have checked existing documentation in [prometheus-operator.dev](https://prometheus-operator.dev)?
+- type: textarea
+  attributes:
+    label: What happened?
+    description: A concise description of what you're experiencing.
+    value: |
+      ## Description
+
+      ## Steps to Reproduce
+
+      ## Expected Result
+
+      ## Actual Result
+
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Prometheus Operator Version
+    description: |
+      Provide the prometheus-operator version used
+      This will be automatically formatted into code, so no need for backticks.
+    placeholder: |
+        - Prometheus Operator version (please provide operator version and not kube-prometheus/helm chart version):
+          ```console
+          Try kubectl -n <namespace> describe deployment/prometheus-operator
```

Comment:
```suggestion
          Try kubectl get deployments.apps -n <namespace> prometheus-operator -o jsonpath='{.spec.template.spec.containers[0].image}'
```

Reply:
Not sure about this actually, maybe we want the whole operator spec instead of just the version?

Reply:
I guess most cases the version is enough 🤔

Reply:
The full deployment spec gives us the arguments list + the resource requests/limits which are useful info.